builds/psvita/Makefile

```diff
@@ -0,0 +1,53 @@
+TARGET		:=	EasyRPG
+BUILD		:=	build
+SOURCES		:=	../../src
+DATA		:=	data
+
+APP_TITLE   	:=	EasyRPG Player
+APP_ID		:=	ERPG00001
+
+CFILES		:=	$(foreach dir,$(SOURCES), $(wildcard $(dir)/*.c))
+CPPFILES   := $(foreach dir,$(SOURCES), $(wildcard $(dir)/*.cpp))
+BINFILES := $(foreach dir,$(DATA), $(wildcard $(dir)/*.bin))
+OBJS     := $(addsuffix .o,$(BINFILES)) $(CFILES:.c=.o) $(CPPFILES:.cpp=.o) 
+
+LIBS = -lvorbisfile -logg -lvorbis -lspeexdsp -lsndfile \
+	-lvita2d -llcf -licuuc -licui18n -licudata -lpixman-1 \
+	-lSceKernel_stub -lSceDisplay_stub -lSceGxm_stub	\
+	-lSceSysmodule_stub -lSceCtrl_stub -lm \
+	-lScePgf_stub -ljpeg -lfreetype -lc \
+	-lScePower_stub -lSceCommonDialog_stub -lpng16 -lz \
+	-lmpg123 -lSceAudio_stub
+
+		
+PREFIX  = arm-vita-eabi
+CC      = $(PREFIX)-gcc
+CXX      = $(PREFIX)-g++
+CFLAGS  = -Wl,-q -O3 -DPSP2 -DHAVE_LIBSPEEXDSP \
+		-DSUPPORT_AUDIO -DHAVE_LIBSNDFILE \
+		-DHAVE_OGGVORBIS -DHAVE_MPG123 -DWANT_FMMIDI=1 \
+		-I$(VITASDK)/arm-vita-eabi/include/liblcf \
+		-I$(VITASDK)/arm-vita-eabi/include/pixman-1
+CXXFLAGS  = $(CFLAGS) -fno-exceptions -std=gnu++11
```

Comment:
For better cooperation with our PR builder I ask you to add some lines here (they won't interfere with yours so is fine ;))

```
Before PREFIX:

ifeq ($(strip $(LIBLCF_DIR)),)
LIBLCF_DIR := $(TOOLCHAIN_DIR)
endif

To CFLAGS add:
		-I$(TOOLCHAIN_DIR)/include \
		-I$(LIBLCF_DIR)/include/liblcf \
		-I$(TOOLCHAIN_DIR)/include/pixman-1 \
		-L$(TOOLCHAIN_DIR)/lib \
		-L$(LIBLCF_DIR)/lib

To CXXFLAGS add (should make the file a bit smaller, we don't need RTTI):
		-fno-rtti